Makefile.work

```diff
@@ -14,6 +14,14 @@
 #  * USERNAME: Desired username -- default at rules/config
 #  * PASSWORD: Desired password -- default at rules/config
 #  * KEEP_SLAVE_ON: Keeps slave container up after building-process concludes.
+#  *                Please note that with current stretch build structure,
+#  *                user of KEEP_SLAVE_ON feature will have to be concious
+#  *                about which docker to stay on.
```

Comment:
Maybe we should say "stay running" to coincide with Docker terminology instead of "stay on"?

Reply:
Thanks, changed now.